wemake_python_styleguide/visitors/tokenize/statements.py

```diff
@@ -256,3 +259,194 @@ def _post_visit(self) -> None:
             self._check_individual_line(
                 line_tokens, previous_line_token, next_line_token,
             )
+
+
+@final
+@dataclass
+class InconsistentComprehensionContext(object):
+    r"""Context for individual bracket enclosures (i.e. [],\{\}, or ())."""
+
+    def __init__(self) -> None:
+        r"""
+        Initializes context for a given bracket enclosure (i.e. [],\{\}, or ()).
+
+        Helper class for InconsistentComprehensionVisitor which stores context
+        for the current (potential) comprehension we are in. Combined with a
+        stack to enable support for nested comprehensions.
+
+        is_comprehension:
```

Comment:
NIP: I am not familiar with this docstring style, but most of these variable names in a docstring are missing `_` prefix. 

```suggestion
        _is_comprehension:
```

Reply:
Good catch. Fixed